app/graphql/resolvers/survey_resolver.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class SurveyResolver < BaseResolver
+    description 'Survey list'
+
+    type [Types::SurveyType], null: false
+
+    def resolve
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/59338f029e412186a0a6dc405561dc7f2a762a99/app/graphql/resolvers/survey_resolver.rb#L9"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>